agent.go

```diff
@@ -200,6 +206,39 @@ func (c *container) getProcess(execID string) (*process, error) {
 	return proc, nil
 }
 
+func (s *sandbox) setSandboxStorage(path string) {
```

Comment:
It's better to return a `bool` here to indicate that if the storage is a new one or an old one. Given such information, the caller would be able to tell if it needs to mount to storage or not, instead of calling `GetRefCount()` again.